### DISCUSSION TITLE
Can't install plugin - isskinu.dll problem

### DISCUSSION THREAD

**Discussion by surkor** [apohl79/audiogridder#1255] [Help]
Hello everyone. I have problem when install some plugins. 

<img width="1440" alt="Screenshot 2023-08-01 at 10 13 01" src="https://github.com/apohl79/audiogridder/assets/141119356/bcd0358f-9b3e-471f-98c1-46c511564de0">

I reinstall microsoft net. framework, reinstall microsoft redistributable c++, download and put manually isskinu.dll, but problem is still exist. 

Please, help

**Comment by williamcorney** [comment]
You need mfc42 for that.  You can install with command,  winetricks mfc42

Oh yeah just to mention if you applying this to a crossover bottle. You will prob need to install wine stable
Winetricks is installed with home brew  

youll need a command like this … WINEPREFIX="/Users/surkor/Library/Application Support/CrossOver/Bottles/REPLACEME" winetricks mfc42
